### ISSUE TITLE
.apk version of app does not work with v1.0.2

### ISSUE THREAD

**Issue by dreautall** [dreautall/waterfly-iii#367]
The .apk version of the app currently does not work. It tries to use the embedded `cronet_http` (HTTP client library), but unfortunately there seems to be a bug (see https://github.com/dart-lang/http/issues/1179). I have removed the .apks from the v1.0.2 release for now due to this.

The Google Play version is not affected, as it uses the Google Play Services `cronet_http` version.

@IzzySoft: not sure if it also affects the apk in your repo. If you're building the APK yourself I guess it also uses the Google Play Services `cronet_http`, as you need to pass extra build flags to **not** use it. Once this ticket is resolved (I'll ping you then again) I suggest to use the extra build flags to not have any Google Play Services dependencies in your build.

**Comment by IzzySoft** [comment]
Thanks for the ping! I'm using the APK provided by you. So once you've fixed the issue and provided a new release with the corresponding APK, it will be fixed automatically when my updater pulls that.

If you want me to remove v1.0.2 for the time being, that could be done – though I then had to disable automated updates for the app or else it would be pulled back in on the next run, so I'd need another ping when I can re-enable it. I can also "demote" the app from daily to monthly update checks, but then v1.0.2 might be pulled back in when the next month starts should there be no newer release by then. Let me know which action you want me to take (if any).

**Comment by IzzySoft** [comment]
> I removed the provided apks from the release page - so does it then just do nothing?

Updater checks releases for *new* versions and *adds* them to the repo. It doesn't check for disappeared ones.

> please stop updates for now, I'll ping you once the upstream issue is fixed

I've just checked, looks like v1.0.2 never arrived here. I see the tag is just 4h old, which perfectly explains that (updater just runs once per day, and didn't run yet after the tag was created). As the APK isn't attached to the tag (anymore; I've missed that part), no action needed from my end: it's not yet here, and not there anymore to be pulled in, so it cannot show up here accidentally. And once you have a ready-to-go APK available, it will be pulled in again.

**TL;DR:** all fine here, and you can "just go on" :smiley:

Thanks for the ping to make sure all is fine – and all the best for getting the issue tackled!

**Comment by IzzySoft** [comment]
Manually triggered an update here when the notification came in. Seems to look fine, thanks! No PlayServices detected.
